js/src/layers/MarkerCluster.js

```diff
@@ -63,6 +65,10 @@ export class LeafletMarkerClusterView extends layer.LeafletLayerView {
   }
 
   create_obj() {
-    this.obj = L.markerClusterGroup();
+    var options = this.get_options();
+    options.data = this.model.get('data');
```

Comment:
What is `this.model.get('data')`?

Reply:
Some remnant code I believe. It can be deleted. my bad